packages/vite/src/node/plugins/importAnalysis.ts

```diff
@@ -466,6 +469,10 @@ export function importAnalysisPlugin(config: ResolvedConfig): Plugin {
               hasEnv = true
             }
             return
+          } else if (templateLiteralRE.test(rawUrl)) {
+            if (!rawUrl.includes('${') && !rawUrl.includes('}')) {
```

Comment:
```suggestion
            if (!(rawUrl.includes('${') && rawUrl.includes('}'))) {
```
I think this is better because it won't match `foo}b`.